auth0/v3/management/email_templates.py

```diff
@@ -0,0 +1,65 @@
+from .rest import RestClient
+
+
+class EmailTemplates(object):
+
+    """Auth0 email templates endpoints
+
+    Args:
+        domain (str): Your Auth0 domain, e.g: 'username.auth0.com'
+
+        token (str): Management API v2 Token
+
+        telemetry (bool, optional): Enable or disable Telemetry
+            (defaults to True)
+    """
+
+    def __init__(self, domain, token, telemetry=True):
+        self.domain = domain
+        self.client = RestClient(jwt=token, telemetry=telemetry)
+
+    def _url(self, id=None):
+        url = 'https://%s/api/v2/email-templates' % self.domain
```

Comment:
Just curious ... is this `sprintf`-type replacement more common than string interpolation in Python? Or is this just a convention for this SDK?

Reply:
I don't know the pros/cons. I'd have used regular string concatenation but I'm copy-pasting another entity basic constructor here, so I'd say this is "convention for this SDK" :trollface: